smartsim/_core/_cli/validate.py

```diff
@@ -155,12 +157,12 @@ def test_install(
         logger.info("Verifying Tensor Transfer")
         client.put_tensor("plain-tensor", np.ones((1, 1, 3, 3)))
         client.get_tensor("plain-tensor")
-        if with_tf:
-            logger.info("Verifying TensorFlow Backend")
-            _test_tf_install(client, location, device)
         if with_pt:
             logger.info("Verifying Torch Backend")
             _test_torch_install(client, device)
+        if with_tf:
```

Comment:
If this fixes the problem we found, could you move the Tensorflow test to the very end?

Reply:
absolutely!!